tests/functional/test_generatecliskeleton.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import json
+
+from awscli.testutils import BaseAWSCommandParamsTest
+
+
+class TestGenerateCliSkeletonOutput(BaseAWSCommandParamsTest):
+    def test_generate_cli_skeleton_output(self):
+        cmdline = 'ec2 describe-regions --generate-cli-skeleton output'
+        stdout, _, _ = self.run_cmd(cmdline)
+        # The format of the response should be a json blob with the
+        # following structure:
+        # {
+        #     "Regions": [
+        #         {
+        #             "RegionName": "RegionName",
+        #             "Endpoint": "Endpoint"
+        #         }
+        #     ]
+        # }
+        #
+        # We assert only components of the response in case members
+        # are added in the future that would break an exactly equals
+        # assertion
+        skeleton_output = json.loads(stdout)
+        self.assertIn('Regions', skeleton_output)
+        self.assertEqual(
+            skeleton_output['Regions'][0]['RegionName'], 'RegionName')
+        self.assertEqual(
+            skeleton_output['Regions'][0]['Endpoint'], 'Endpoint')
```

Comment:
I left a comment above, but I am trying to avoid this test breaking if ec2 ever adds another member to its DescribeRegions output. Does that make sense? I can also just use the entire response if we feel comfortable that the operation will not change, but there is no guarantees.